src/backend/aspen/api/views/samples.py

```diff
@@ -73,7 +73,8 @@ async def list_samples(
         selectinload(Sample.uploaded_by),
         selectinload(Sample.collection_location),
         selectinload(Sample.accessions),
-    )
+        selectinload(Sample.pathogen),
+    ).filter(Sample.pathogen_id == pathogen.id)
```

Comment:
This PR looks good! Only question is whether this `list` endpoint behaves reasonably well when `sample.pathogen_id` is null

Reply:
oh that's right we will have some samples without pathogen in the interim.. I think the best case here would be to return an exception?

Reply:
ha, we can't return an exception since we want our endpoints to keep working properly until we update all samples!

Returning 'pathogen: null` in the response is fine for the interim, but I want to make sure that samples without pathogens are still listed.

Reply:
so actually if we do have a null pathogen_slug the `get_pathogen_slug` will return `SC2` as a default, and in `get_pathogen` we `raise ex.BadRequestException("Invalid pathogen slug")` if no pathogen was found, so i think we're set on this and won't get any wierd edge cases. I'll add another test to verify though